README.md

```diff
@@ -311,6 +311,23 @@ const clientOpts = await connector.getOptions({
   authType: AuthTypes.IAM,
 });
 ```
+## Support policy
+
+### Major version lifecycle
+
+This project uses [semantic versioning](https://semver.org/), and uses the
+following lifecycle regarding support for a major version:
+
+**Active** - Active versions get all new features and security fixes (that
+wouldn’t otherwise introduce a breaking change). New major versions are
+guaranteed to be "active" for a minimum of 1 year.
+
+**Deprecated** - Deprecated versions continue to receive security and critical
+bug fixes, but do not receive new features. Deprecated versions will be
+supported for 1 year.
+
+**Unsupported** - Any major version that has been deprecated for >=1 year is
+considered unsupported.
 
 ## Supported Node.js Versions
```

Comment:
should this be h3?

Reply:
in both Python and Go connectors they're h2

Reply:
Just put PRs to adjust them. Supported versions falls under support policy

Reply:
sounds good! 👍